content/en/faq/applications/elasticsearch.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Elasticsearch Configuration
```

Comment:
FAQ entries should be in the form of a question

Reply:
Done.